index.js

```diff
@@ -1,7 +1,90 @@
-// Iteration 1: Names and Input
-
-
-// Iteration 2: Conditionals
-
-
-// Iteration 3: Loops
+//1.ITERATION//
+let hacker1 = "Alberto";
+console.log("The driver's name is", hacker1);
+
+let hacker2 = "Juan";
+console.log("the navigator's name is", hacker2);
+
+//2.CONDITIONALS//
+let longitudHacker1 = hacker1.length;
+let longitudHacker2 = hacker2.length;
+
+if (longitudHacker1 > longitudHacker2) {
+  console.log("The driver has the longest name, it has " + longitudHacker1 + " characters.");
+}
+else if (longitudHacker2 > longitudHacker1) {
+  console.log("It seems that the navigator has the longest name, it has " + longitudHacker2 + " characters.");
+}
+else {
+  console.log("Wow, you both have equally long names, " + longitudHacker1 + " characters!.");
+}
+
+//3.LOOPS//
+let result = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  result += hacker1[i].toUpperCase();

```

Comment:
Aqui me falta un espacio entre cada letra! :D

```suggestion
  result += `${hacker1[i].toUpperCase()} `;

```

Pero si la pones al final de cada letra entonces la resultante tendrá un espacio al final, se te ocurre como quitarlo???